src/nidcpower/metadata/functions_addon.py

```diff
@@ -84,6 +84,11 @@
     'GetError':                     { 'is_error_handling': True },
 }
 
+# There are some parameters that are needed in the C function call we use under the hood, but that we do not want in the Python API
+functions_remove_from_python_api = {
+    'FetchMultiple':                { 'parameters': { 7: { 'use_in_python_api': False, }, }, },
```

Comment:
Does this get populated correctly at add_all time for things that we don't specify?
For example, in `nifgen.Session.write_waveform` the size should have a value of False.

Reply:
Added auto population for 'len' and 'ivi-dance' size parameters